File: C11/백준/Bronze/10818. 최소， 최대/최소， 최대.c
#include <stdio.h>

int main(void){
	
	int N;
	scanf("%d", &N);
	
	int arr[N];
	
	int max = -1000001;
	int min = 1000001;
	
	for(int i = 0; i < N; i++){
		scanf("%d", &arr[i]);
		if (arr[i] > max) {
            max = arr[i];
        }
        if (arr[i] < min) {
            min = arr[i];
        }
	}
	
	printf("%d %d", min, max);
	
	return 0;
}

File: Java/백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        Scanner s = new Scanner(System.in);

        int N = s.nextInt(); //정수의 개수
        int[] arr = new int[N];

        int max = Integer.MIN_VALUE;
        int min = Integer.MAX_VALUE;

        for (int i = 0; i < N; i++) {
            arr[i] = s.nextInt();
            if (arr[i] > max) {
                max = arr[i];
            }
            if (arr[i] < min) {
                min = arr[i];
            }
        }

        System.out.print(min + " " + max);

        s.close();
    }
}